products/deploymentmanager/api.yaml

```diff
@@ -0,0 +1,177 @@
+# Copyright 2017 Google Inc.
```

Comment:
I'm not sure if anything has changed, but the last time I tried this the magician errored because it checks for the license against a specific license in 2017 lol